pkg/ui/src/util/format.ts

```diff
@@ -136,3 +136,6 @@ export const DurationFitScale = (scale: string) => (nanoseconds: number) => {
   const n = durationUnits.indexOf(scale) ;
   return `${(nanoseconds / Math.pow(1000, n)).toFixed(2)} ${scale}`;
 };
+
+// tslint:disable-next-line: variable-name
```

Comment:
@elkmaster please follow tslint naming recommendations: `DATE_FORMAT` is preferred for a top-level non-function constant.